File: app/src/main/java/com/example/android/pingpongscorekeeper/activities/SettingsActivity.java
package com.example.android.pingpongscorekeeper.activities;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.example.android.pingpongscorekeeper.fragments.SettingsFragment;

//TODO: Why is AppCompactActivity and not Activity used here?
public class SettingsActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        getFragmentManager().beginTransaction()
                .replace(android.R.id.content, new SettingsFragment())
                .commit();
    }
}

File: app/src/main/java/me/dhivo/android/pingpongmatchtracker/activities/SettingsActivity.kt
package me.dhivo.android.pingpongmatchtracker.activities

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.MenuItem

import me.dhivo.android.pingpongmatchtracker.fragments.SettingsFragment

//TODO: Why is AppCompactActivity and not Activity used here?
class SettingsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)

        fragmentManager.beginTransaction()
                .replace(android.R.id.content, SettingsFragment())
                .commit()
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            android.R.id.home -> {
                finish()
                return true
            }
        }
        return super.onOptionsItemSelected(item)
    }
}
